bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/dependencies/sqlite/users/UsersTableAccessor.java

```diff
@@ -22,19 +22,23 @@ public UsersTableAccessor(SQLiteConfig sqliteConfig) {
 	}
 
 	public User getUserById(int userId) throws SQLException {
-		String sql = "SELECT * FROM users WHERE userId = ? LIMIT 1";
-
 		Connection connection = getDatabaseConnection();
+		String sql = "SELECT id, first_name, last_name, email, password, created_at FROM users WHERE userId = ? LIMIT 1";
```

Comment:
You had it right the first time, `*` is exactly right here. But it's fine in this case. Also I didn't notice initially, but `userId` should be `user_id`